kalite/coachreports/views.py

```diff
@@ -207,7 +209,7 @@ def tabular_view(request, report_type="exercise"):
 
     context.update({
         # For translators: the following two translations are nouns
-        "report_types": (_("exercise"), _("video")),
+        "report_types": ({"value": "exercise", "name":_("exercise")}, {"value": "video", "name": _("video")}),
```

Comment:
Concomitant with change in template above. Allows value and translated value not to be conflated.